src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -1327,6 +1327,55 @@ def getConfigCacheIDs(self):
                 IDs.append(ID)
         return IDs
 
+    def physicsTaskType(self):
+        """
+        _physicsTaskType_
+
+        Return the physics Task type
+        """
+        return getattr(self.data, 'physicsTaskType', None)
+
+    def getStepPhysicsTypes(self):
+        """
+        _getStepPhysicsTypes_
+        Get the physics types for all cmsRun steps
+        """
+        types = []
+        for stepName in self.listAllStepNames(cmsRunOnly=True):
+           stepHelper = self.getStepHelper(stepName)
+           stepType = stepHelper.getPhysicsType()
+           if stepType:
+               types.append(stepType)
+           else:
+               types.append("UNKNOWN")
+        return types
+
+    def setPhysicsTaskType(self):
+        """
+        _setPhysicsType_
```

Comment:
I don't know what the reason for this string `_setPhysicsType_`, maybe it was a legacy for Sphinx or another auto-documentation tool.
In any case, I would suggest to stick with what we have in the contribution guidelines:
https://github.com/dmwm/WMCore/blob/master/CONTRIBUTING.rst#project-docstrings-best-practices

even if that makes the module's docstring inconsistent. Provided we are moving forward, even in baby steps, eventually we will converge and have something more homogeneous.

Reply:
@amaltaro Done!